src/analyses/baseInvariant.ml

```diff
@@ -100,6 +100,25 @@ struct
         if M.tracing then M.tracel "inv" "st from %a to %a" D.pretty st D.pretty r;
         r
       )
+    | Mem (Lval lv), NoOffset ->
```

Comment:
Why is it necessary to limit this to `NoOffset`? Seems like it shouldn't really matter whether the pointer in in a struct field or something.

Reply:
I have generalized this now.